src/Sulu/Bundle/TagBundle/Resources/translations/admin.de.json

```
@@ -1,5 +1,9 @@
 {
     "sulu_tag.tags": "Tags",
     "sulu_tag.tag": "Tag",
-    "sulu_tag.no_commas_allowed": "In diesem Feld sind keine Beistriche erlaubt"
+    "sulu_tag.no_commas_allowed": "In diesem Feld sind keine Beistriche erlaubt",
+    "sulu_activity.text.tags.created": "{userFullName} hat den Tag \"{resourceTitle}\" erstellt",
+    "sulu_activity.text.tags.modified": "{userFullName} hat den Tag \"{resourceTitle}\" geändert",
+    "sulu_activity.text.tags.removed": "{userFullName} hat den Tag \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.tags.merged": "{userFullName} hat den Tag \"{context_sourceTagName}\" in den Tag \"{resourceTitle}\" vereinigt"
```

```suggestion
    "sulu_activity.text.tags.merged": "{userFullName} hat die Tags \"{context_sourceTagName}\" und \"{resourceTitle}\" zusammengeführt"
```

I think `zusammengeführt` sounds a little bit better 🙈